geoportailv3/static/js/themesservice.js

```diff
@@ -125,6 +125,13 @@ app.Themes.prototype.getBgLayers = function() {
           goog.asserts.assert('imageType' in item);
           var layer = this.getWmtsLayer_(item['name'], item['imageType']);
           layer.set('metadata', item['metadata']);
+
+          if (goog.object.containsKey(item['metadata'], 'attribution')) {
+            var source = layer.getSource();
+            source.setAttributions(
+              item['metadata']['attribution']
```

Comment:
Do you know how OL3 reacts if a same text is provided to different layers ? Is the same text displayed displayed many times ?